modules/ocf/manifests/systemd/timer.pp

```diff
@@ -0,0 +1,48 @@
+define ocf::systemd::timer(
+    $minute = '*',
+    $hour   = '*',
+    $day    = '*',
+    $month  = '*',
+    $year   = '*',
+    # For reasonable second precision,
+    # AccuracySec will need to be set
+    $on_calendar = "${year}-${month}-${day} ${hour}:${minute}:00",
+    $timer_template = undef,
+    $timer_source = undef,
+    $service_template = undef,
+    $service_source = undef,
+    $exec_user = 'nobody',
+    $exec_start = '',
+  ) {
+
+  file {
+    default:
+      owner => root,
+      group => root,
+      mode  => '0644';
+
+    "/etc/systemd/system/${title}.timer":
+      content => template($timer_template),
+      source  => $timer_source;
+
+    "/etc/systemd/system/${title}.service":
```

Comment:
I would suggest using `ocf::systemd::service` here instead.

Reply:
Yeah, I was thinking the same thing. Declare the service file first and ocf::systemd::service will create the timer file and start the service.

Reply:
? I was saying that instead of creating a raw service file here, you can make use of the existing `ocf::systemd::service`.

Reply:
No. We don't want to use that because ocf::systemd::service starts a service for us, which we do not want. The timer should start the service for us. From the systemd.timer manpage:

>For each timer file, a matching unit file must exist, describing the unit to activate when
>the timer elapses. By default, a service by the same name as the timer (except for the
>suffix) is activated.

Reply:
Ah, I see. Are you suggesting to set it up like this?

```enable => false,
    ensure => 'stopped',
```
If so, I'm on board with this idea.

Reply:
> No. We don't want to use that because ocf::systemd::service starts a service for us, which we do not want.

Ah right, never mind then. Nah, we don't want to ensure stopped, that would stop the service even when we wanted it to run.